### PR TITLE
add tools page

### PR DESCRIPTION
**Pull request by rawxfish**
very rough initial version for #4 

Screen Shot 2019-12-15 at 2 04 51 PM

### CODE REVIEW

File: hunts/templates/tools.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <h2>Word Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>https://quipqiup.com/</li>
+            <li>https://nutrimatic.org/</li>
+            <li>https://www.onelook.com/</li>
+            <li>https://www.thewordfinder.com/anagram-solver/</li>
+            <li>https://www.highlightpress.com.au/</li>
+            <li>http://www.npinsker.me/puzzles/crossword</li>
```
Comment (maximized) [inline hunts/templates/tools.html]: Can we include a description, either as the link text or before or after the link? For example, it is important to note that Nathan's link doesn't help with crossword clues, but parses crosswords

File: hunts/templates/tools.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Logic Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li>Ask Max</li>
```
Comment (maximized) [inline hunts/templates/tools.html]: Grid Puzzles - https://en.wikipedia.org/wiki/Nikoli_(publisher)

Comment (erwa) [reply hunts/templates/tools.html]: And a good reference list of logic puzzles with example pictures and rules is: http://www.cross-plus-a.com/puzzles.htm

**Comment by rawxfish** [comment]
Screen Shot 2019-12-15 at 2 32 29 PM

File: hunts/templates/tools.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <div class="col-10">
+        <h2>Word Puzzles</h2>
+    </div>
+    <div class="col-10">
+        <ul>
+            <li><a href="https://quipqiup.com/">quipqiup</a></li>
+            <li><a href="https://nutrimatic.org/">nutrimatic</a></li>
+            <li><a href="https://www.onelook.com/">OneLook</a></li>
+            <li><a href="https://www.thewordfinder.com/anagram-solver/">Anagram Solver</a></li>
+            <li><a href="https://www.highlightpress.com.au/">Cryptics Reference</a></li>
```
Comment (erwa) [inline hunts/templates/tools.html]: Other useful cryptic references:

http://bestforpuzzles.com/cryptic-crossword-dictionary -> has A-Z dictionary
http://cryptics.wikia.com/wiki/List_of_anagram_indicators

File: hunts/templates/tools.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+    <div class="col-10">
+        <ul>
+            <li><a href="https://oeis.org/">The On-Line Encyclopedia of Integer Sequences</a></li>
+            <li><a href="https://www.paulschou.com/tools/xlate/">Encodings</a></li>
+        </ul>
+    </div>
+
+    <div class="col-10">
+        <h2>Technical</h2>
```
Comment (erwa) [inline hunts/templates/tools.html]: Solving jigsaw in GIMP: http://nastyissues.blogspot.com/2019/11/solving-jigsaw-puzzle-in-gimp.html

**Pull request merged**